diamond-api/src/render.rs

```diff
@@ -104,6 +105,12 @@ impl FromStr for RenderFormat {
     }
 }
 
+impl Display for RenderFormat {
+    fn fmt(&self, f: &mut Formatter) -> Result<(), std::fmt::Error> {
+        write!(f, "{:?}", self)
```

Comment:
This implementation will print capitalized values (derived Debug).

Display trait also defines `to_string()` method (via ToString trait). There will be asymmetry because `.from_str()` expect lowercased input now.

Reply:
Adding .to_ascii_lowercase()

Reply:
He-he. Ok